src/main/java/nextstep/subway/favorite/application/FavoriteService.java

```diff
@@ -0,0 +1,68 @@
+package nextstep.subway.favorite.application;
+
+import nextstep.subway.exception.NotFoundException;
+import nextstep.subway.favorite.domain.Favorite;
+import nextstep.subway.favorite.domain.FavoriteRepository;
+import nextstep.subway.favorite.dto.FavoriteRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import nextstep.subway.member.domain.Member;
+import nextstep.subway.member.domain.MemberRepository;
+import nextstep.subway.member.exception.MemberExceptionCode;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.StationRepository;
+import nextstep.subway.station.exception.StationExceptionCode;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class FavoriteService {
+
+    private final MemberRepository memberRepository;
+    private final FavoriteRepository favoriteRepository;
+    private final StationRepository stationRepository;
+
+    public FavoriteService(MemberRepository memberRepository, FavoriteRepository favoriteRepository,
+            StationRepository stationRepository) {
+        this.memberRepository = memberRepository;
+        this.favoriteRepository = favoriteRepository;
+        this.stationRepository = stationRepository;
+    }
+
+    @Transactional
+    public FavoriteResponse createFavorite(Long memberId, FavoriteRequest request) {
+        Member member = findMemberId(memberId);
+        Station source = findStationById(request.getSource());
+        Station target = findStationById(request.getTarget());
+        Favorite favorite = favoriteRepository.save(request.toFavorite(member, source, target));
```

Comment:
```suggestion
        Favorite favorite = favoriteRepository.save(new Favorite(member, source, target));
```
`request`랑 관계없이 `Favorite`를 만들고있는데 `Favorite`의 생성자를 통해 만들어도 되지 않을까요?